samples/peripheral/802154_phy_test/sample.yaml

```diff
@@ -9,4 +9,22 @@ tests:
       - nrf21540dk_nrf52840
       - nrf5340dk_nrf5340_cpunet
     platform_allow: nrf52840dk_nrf52840 nrf21540dk_nrf52840 nrf5340dk_nrf5340_cpunet
-    tags: ci_build
+    tags: ci_rs_build
+  sample.peripheral.802154_phy_test_auto:
```

Comment:
add antenna diversity in sample name, maybe `sample.peripheral.802154_phy_test.ant_div_mode_auto`

Reply:
Moreover Antenna diversity is valid only for nrf21540dk platform that has phisicall two antennas
nrf52840dk is incorrect in this situation (valid only if FEM EK is added)

Reply:
fixed :)